README.md

```diff
@@ -14,7 +14,7 @@
 7. CSS должен быть написан с&nbsp;использованием препроцессоров (LESS/Sass/Stylus). Желательно использование систем сборки (Grunt/Gulp) и&nbsp;построцессоров (PostCSS/Autoprefixer).
 8. Проверка и&nbsp;оптимизация скорости загрузки: https://github.com/ihorzenich/WebPerformanceChecklist
 9. Поддержка Retina
-10. Наличие Win/Mac/Linux-аналогов шрифтов
+10. Наличие Windows , Mac or Linux users -аналогов шрифтов
```

Comment:
No need to just replace abbr. Also `Linux users` - is mistake, we are talking about fonts here.